README.md

```diff
@@ -396,8 +396,8 @@ Some standouts:
 
 | Hardware   | Instructions                                                                                                    |
 |------------|-----------------------------------------------------------------------------------------------------------------|
-| CPU        | `pip install -U "jax[cpu]"`                                                                                       |
-| NVIDIA GPU on x86_64 | `pip install -U "jax[cuda12]"`        |
+| CPU        | `pip install -U "jax[cpu]"`                                                                                     |
```

Comment:
Do we need cpu anymore?

Reply:
I emptied it out but left the extra around for the moment.

Removed it from the docs, mostly, though.